packages/components/time-picker/src/time-picker-com/basic-time-spinner.vue

```diff
@@ -231,13 +231,31 @@ const scrollDown = (step: number) => {
   const label = currentScrollbar.value!
   let now = unref(timePartials)[label]
   const total = currentScrollbar.value === 'hours' ? 24 : 60
-  now = (now + step + total) % total
-
+  now = findNextUnDisabled(label, now, step, total)
   modifyDateField(label, now)
   adjustSpinner(label, now)
   nextTick(() => emitSelectRange(label))
 }
-
```

Comment:
Maybe you can refer to the format of the current file and keep the newlines at both ends of the function.